From c184c74fdad63385e249b8448efd3a73e2b212c5 Mon Sep 17 00:00:00 2001
From: Xiao Wang <jasowang@redhat.com>
Date: Fri, 10 Aug 2012 09:03:48 -0300
Subject: [RHEL6 qemu-kvm PATCH 8/9] e1000: introduce bit for debugging PHY
 emulation

RH-Author: Xiao Wang <jasowang@redhat.com>
Message-id: <1344589429-3229-9-git-send-email-jasowang@redhat.com>
Patchwork-id: 40682
O-Subject: [RHEL6.4 qemu-kvm 8/9] e1000: introduce bit for debugging PHY emulation
Bugzilla: 607510 819915
RH-Acked-by: Michael S. Tsirkin <mst@redhat.com>
RH-Acked-by: Alex Williamson <alex.williamson@redhat.com>
RH-Acked-by: Vlad Yasevich <vyasevic@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=607510

Signed-off-by: Jason Wang <jasowang@redhat.com>
Signed-off-by: Michael S. Tsirkin <mst@redhat.com>
(cherry picked from commit f9c1cdf4928e6c036a6373e683d1774129a10311)
---
 hw/e1000.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

Signed-off-by: Eduardo Habkost <ehabkost@redhat.com>
---
 hw/e1000.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/hw/e1000.c b/hw/e1000.c
index 0c50e1d..75c2974 100644
--- a/hw/e1000.c
+++ b/hw/e1000.c
@@ -38,7 +38,7 @@ enum {
     DEBUG_GENERAL,	DEBUG_IO,	DEBUG_MMIO,	DEBUG_INTERRUPT,
     DEBUG_RX,		DEBUG_TX,	DEBUG_MDIC,	DEBUG_EEPROM,
     DEBUG_UNKNOWN,	DEBUG_TXSUM,	DEBUG_TXERR,	DEBUG_RXERR,
-    DEBUG_RXFILTER,	DEBUG_NOTYET,
+    DEBUG_RXFILTER,     DEBUG_PHY,      DEBUG_NOTYET,
 };
 #define DBGBIT(x)	(1<<DEBUG_##x)
 static int debugflags = DBGBIT(TXERR) | DBGBIT(GENERAL);
-- 
1.7.11.2

